### DISCUSSION TITLE
Is it the expected behavior to have the event handlers invoked sequentially?

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#10016] [Q&A]
I have been following the instructions in the guide here https://discordjs.guide/creating-your-bot/event-handling.html#resulting-code.

Is it the expected behavior to have the event handlers invoked sequentially? (I mean one by one, in FIFO fashion)

Thank you for any insights.

**Comment by Jiralite** [comment]
Duplicate of https://github.com/discordjs/guide/discussions/1531.
